app/api/bedrock.ts

```
@@ -0,0 +1,267 @@
+      throw new Error(`Invalid JSON in request body: ${e}`);
+    }
+    // console.log(
+    //   "[Bedrock Request] original Body:",
+    //   JSON.stringify(bodyJson, null, 2),
+    // );
+
+    // Extract tool configuration if present
+    let tools: any[] | undefined;
+    if (bodyJson.tools) {
+      tools = bodyJson.tools;
+      delete bodyJson.tools; // Remove from main request body
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)